CMakeLists.txt

```diff
@@ -923,6 +924,9 @@ endif(DEFINED WITH_GIGEAPI)
 
 if(DEFINED APPLE)
   status("    AVFoundation:"   HAVE_AVFOUNDATION   THEN YES                                        ELSE NO)
+  if(WITH_COREIMAGE OR HAVE_COREIMAGE)
+  status("    CoreImage:"      HAVE_COREIMAGE      THEN "YES (deprecated)"                         ELSE NO)
```

Comment:
Why "deprecated"?
Looks like it is enabled on modern environments: `XCODE_VERSION VERSION_GREATER 7.0`